### PR TITLE
Package name changed to `TimeFrames`.

### PR DESCRIPTION
**Pull request by codetalker7**
As suggested in https://github.com/xKDR/TSx.jl/issues/66, this PR changes the package name to `TimeFrames` and the struct name to `TimeFrame`.

### CODE REVIEW

File: Benchmark.md [hunk 1]
```diff
@@ -26,9 +26,9 @@ Rows: 500
 
 Columns: timestamp, Open, High, Low, Close
 
-## Creating TSx object from DataFrame
+## Creating TimeFrames object from DataFrame
```
Comment (ayushpatnaikgit) [inline Benchmark.md:29]: Should be it “Creating a TimeFrame” object?

Comment (codetalker7) [reply Benchmark.md:29]: Yes, I think it should be that. I'll make the change.

**Comment by ayushpatnaikgit** [comment]
I think it's good to merge

**Comment by chiraganand** [comment]
> ```
> ==========================================
> - Coverage   89.40%   88.91%   -0.49%     
> ==========================================
>   Files          19       19              
>   Lines         368      370       +2     
> ==========================================
>   Hits          329      329              
> - Misses         39       41       +2     
> ```

How come the coverage is getting reduced after renaming? There are two lines which are getting missed.

**Comment by codetalker7** [comment]
We can try running the Codecov job again to verify if this is indeed the case.

File: src/TimeFrame.jl [hunk 2]
```diff
@@ -58,8 +58,8 @@ julia> df = DataFrame(x1 = random(10))
    9 │ 0.26864
   10 │ 0.108871
 
-julia> ts = TS(df)   # generates index
-(10 x 1) TS with Int64 Index
+julia> ts = TimeFrame(df)   # generates index
```
Comment (chiraganand) [inline src/TimeFrame.jl:61]: We should also change `ts` to `tf` to go with the latest convention.

**Comment by chiraganand** [comment]
> > > ```
> > > ==========================================
> > > - Coverage   89.40%   88.91%   -0.49%     
> > > ==========================================
> > >   Files          19       19              
> > >   Lines         368      370       +2     
> > > ==========================================
> > >   Hits          329      329              
> > > - Misses         39       41       +2     
> > > ```
> > 
> > 
> > How come the coverage is getting reduced after renaming? There are two lines which are getting missed.
> 
> Yeah this is a bit weird. Codecov is not showing which lines are missed either. Could it be because of some test which is uses randomization without a random seed?

Usually, when Codecov doesn't show the diff then there is something wrong with the history of codecov CI runs. I saw there were two failed CI runs (macOS timing out) in the master a few commits behind, I have restarted them. Let us see if that fixes it.

**Comment by chiraganand** [comment]
> > > > ```
> > > > ==========================================
> > > > - Coverage   89.40%   88.91%   -0.49%     
> > > > ==========================================
> > > >   Files          19       19              
> > > >   Lines         368      370       +2     
> > > > ==========================================
> > > >   Hits          329      329              
> > > > - Misses         39       41       +2     
> > > > ```
> > > 
> > > 
> > > How come the coverage is getting reduced after renaming? There are two lines which are getting missed.
> > 
> > 
> > Yeah this is a bit weird. Codecov is not showing which lines are missed either. Could it be because of some test which is uses randomization without a random seed?
> 
> Usually, when Codecov doesn't show the diff then there is something wrong with the history of codecov CI runs. I saw there were two failed CI runs (macOS timing out) in the master a few commits behind, I have restarted them. Let us see if that fixes it.

Re-run of the codecov job in this PR fixed the Codecov output but the full output doesn't show any file which has decreased in test coverage only the top level percentage is shown to be decreased by 0.48%. Don't know why this is happening but guess we can leave this investigation for later.

**Pull request merged**